src/validators/mod.rs

```diff
@@ -205,7 +205,7 @@ impl SchemaValidator {
 
     pub fn __repr__(&self, py: Python) -> String {
         format!(
-            "SchemaValidator(name={:?}, validator={:#?}, slots={:#?})",
+            "SchemaValidator(title={:?}, validator={:#?}, slots={:#?})",
```

Comment:
Do you mind if I sneak this in here? I'm guessing the struct field changed name at some point and this wasn't updated. I can put it in a separate PR or just remove and leave as is, just had this sitting around in my diff when I opened the pydantic-core repo and figured I could maybe just bring the change in with this..

Reply:
Nevermind, need to update other tests; I'll make it a separate pr..